Gemfile

```diff
@@ -79,11 +79,12 @@ gem 'therubyracer'
 gem 'thin'
 gem 'timecop'
 gem 'tinymce-rails'
-gem 'uglifier'
```

Comment:
Have you tested if application is working fine without the uglifier gem? I see it was being used inside tree_display.jsx file.

Reply:
The application seems to load fine, but I need someone from the development team to ensure that no core functionality is affected. It's majorly used for JS compression, so I don't expect any major effects. We talked about this and you said you were looking into it.